### PR TITLE
add ozone concentration artifact

### PR DESCRIPTION
**Pull request by szy21**
Checklist:
- [x] I created a new folder `$artifact_name`
  - [x] I added a `README.md` in that that folder that
    - [x] describes the data and processing done to it
    - [x] lists the sources of the raw data
    - [x] lists the required citation, licenses
  - [ ] If applicable (e.g., for Creative Commons), I added a `LICENSE` file
  - [x] I added the scripts that retrieve, process, and produce the artifact
  - [x] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [x] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [x] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [x] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- [x] I added a link to the main `README.md` to point to the new artifact

### CODE REVIEW

File: ozone_concentrations/create_artifact.jl [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+             ]
+
+const H_EARTH = 7000.0
+const P0 = 1e5
+const HPA_TO_PA = 100.0
+
+Plvl_inv(P) = -H_EARTH * log(P / P0)
+
+function create_ozone(infile_paths, outfile_path)
+    FT = Float64
```
Comment (szy21) [inline ozone_concentrations/create_artifact.jl:23]: Do we need Float64? The file is pretty big.

Comment (Sbozzolo) [reply ozone_concentrations/create_artifact.jl:23]: There is no engineering restriction coming from the float type of this file. If Float32 is good for science, you can go for it.

File: ozone_concentrations/create_artifact.jl [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    time_ = defVar(ncout, "time", FT, ("time",), attrib = ncin["time"].attrib)
+    time_[:] = Array(ncin["time"])
+
+    defVar(
+        ncout,
+        "vmro3",
+        FT,
+        ("lon", "lat", "z", "time"),
+        attrib = filter(((k, v),) -> k != "_FillValue", ncin["vmro3"].attrib),
```
Comment (Sbozzolo) [inline ozone_concentrations/create_artifact.jl:53]: Maybe add a comment on why we filter out `_FillValue`?

Comment (szy21) [reply ozone_concentrations/create_artifact.jl]: I tried it again without the filter and it works now, so I removed it. (Not sure what happened before, maybe I was using a different version of some packages).

**Pull request merged**